### DISCUSSION TITLE
Determine the Magick.Native version (git commit id) for specific build for Magic.NET

### DISCUSSION THREAD

**Discussion by t246246** [dlemstra/Magick.NET#661] [General] [answered]
Once I asked the same question and get the answer here:
https://github.com/dlemstra/Magick.NET/issues/521

Now, this no longer works because the URL of dev.azure.com is not found, probably need authentication.

Is there a way to determine the ImageMagick version for Magick.NET 7.19.0.0?
src/Magick.Native/Magic.Native.version is 0.2020.606.1430.

Best Wishes,

**Comment by dlemstra** [comment, marked as answer]
Would it help if the commit id was written to this file: https://github.com/dlemstra/Magick.NET/blob/9c11bb0b745b680632b500690234b5fd49807e1e/src/Magick.Native/libraries.md?

Maybe add this?

`Magick.Native (#commit-id) is build with the following libraries:`

**Reply by t246246** [reply]
Thank you.  I'd like to know Magick.Native commit id from Magick.NET release version,
i.e. need to confirm Mgick.NET 7.019.0.1 is using Magick.Native 03c5f4652a3ab29364b9b01dd1dcfcf7ba01eaa9 .

So it would be good idea adding commit id of Magick.Native in libraried.md of Magick.NET

**Reply by dlemstra** [reply]
The commit ID has been added to the libraries.md file.

**Reply by t246246** [reply]
Many thanks.  I confirmed it, which could help those who need to build Magic.NET by themselves.
